server/aurora_mcp/tools_always_on.py

```
@@ -75,34 +75,35 @@ async def _do_trigger_rca(
+
+async def _do_read_memory(api_call: ApiCall, entry_id: str) -> Dict[str, Any]:
+    return truncate_payload(
+        await api_call("GET", f"/api/memory/entries/{entry_id}"),
+        tool_name="read_memory",
     )
 
 
 async def _do_search_runbooks(api_call: ApiCall, query: str, limit: int) -> Dict[str, Any]:
-    kb_call = api_call("POST", "/api/knowledge-base/search",
-                       body={"query": query, "limit": limit})
+    mem_call = api_call("GET", f"/api/memory/entries?category=runbook")
```

Ruff (0.15.18)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)